code/modules/admin/verbs/gimmick_team.dm

```diff
@@ -53,7 +53,9 @@
 			players_to_spawn += candidate
 	else
 		to_chat(src, "Polling candidates...")
-		players_to_spawn = SSghost_spawns.poll_candidates("Do you want to play as \a [O.name]?")
+		var/datum/outfit/temp = new O
+		players_to_spawn = SSghost_spawns.poll_candidates("Do you want to play as \a [temp.name]?")
+		qdel(temp)
```

Comment:
Pretty sure you can get away with this:
```suggestion
		players_to_spawn = SSghost_spawns.poll_candidates("Do you want to play as \a [initial(O.name])?")
```
I tested this and it seemed to give me the proper outfit name. You may want to double check though.